### DISCUSSION TITLE
What contract address does "address" refer to in the subgraph manifest

### DISCUSSION THREAD

**Discussion by zendevil** [graphprotocol/graph-tooling#738] [Q&A]
In the subgraph manifest, there's an address in dataSources[i].kind.source.address. Since there will be multiple contracts deployed in the future of the same abi, both in development and in production on ganache and on mainnet respectively, I don't understand what it means to provide a contract address in this case. What's the point of the contract address and where does it come from?

**Comment by schmidsi** [comment]
Each subgraph is only indexing one blockchain. If you want to deploy the same subgraph on different chains you need to create a new subgraph manifest per chain.
